app/models/manageiq/providers/inventory/persister/builder.rb

```diff
@@ -267,6 +285,12 @@ def evaluate_lambdas_on(values, persister)
           end
         end
       end
+
+      private
+
+      def without_sti?
+        !!@options[:without_sti]
+      end
```

Comment:
For legibility, WDYT about

```ruby
def with_sti?
  !@options[:without_sti]
end
```

This kind of mirrors the setter method `with_sti` and keeps everything positive without having to mess with the option name.